tests/kleros.test.js

```diff
@@ -904,6 +904,6 @@ describe('Kleros', () => {
       expect(disputeData.appealCreatedAt.length).toEqual(1)
       expect(disputeData.appealRuledAt.length).toEqual(1)
     },
-    100000
+    200000
```

Comment:
Why would the tests be taking twice as long? If its really these changes that causes this we should try to find out what is slowing things down

Reply:
I haven't added any new async operations, but I'll look into it.